generated/global-autoconfig-properties/pom.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>generated-parent</artifactId>
+        <version>3.5.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>global-spring-autoconfig-properties</artifactId>
+    <packaging>pom</packaging>
```

Comment:
I think only the parent pom needs to be marked as `<packaging>pom</packaging>` so maven won't produce further artifacts. Submodules should have the default packaging as jar. For reference, checkout submodules under`spring-cloud-gcp-samples`.